### PR TITLE
Mods dev server

### PR DESCRIPTION
**Pull request by objerke**
Replace the current combination of `live-server` and `development-server.js` with `@tibco/spotfire-mods-dev-server` in all examples. Also add the new tools/mods-dev-server folder which contains the source code of the new package.

### CODE REVIEW

File: tools/mods-dev-server/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+With [Spotfire Mods](https://tibcosoftware.github.io/spotfire-mods/), you can include custom visualizations in your Spotfire® applications much faster by integrating JavaScript visualizations. Create your own visualizations that look and feel like the native Spotfire visualizations, and that leverage the built-in capabilities of Spotfire.
+
+This package is a simple web server with the aim to speed up mod development. It mimics the behavior of the Spotfire Mods sandbox. It is the default server used by the developer examples in the [Spotfire Mods GitHub repository](https://github.com/TIBCOSoftware/spotfire-mods).
+
+The development server is based on the [live-server](https://www.npmjs.com/package/live-server) package with an additional middleware for caching and CSP headers.
+
+Invoke the following command from a `package.json` script to start the server:
+
+```bash
+mods-dev-server src
```
Comment (ebrandin) [inline tools/mods-dev-server/README.md:12]: This assumes the name of the source folder '\src'.

Comment (objerke) [reply tools/mods-dev-server/README.md:12]: Yes, you can pass any folder name as an argument to the server.

Comment (ebrandin) [inline tools/mods-dev-server/README.md:12]:
```suggestion
mods-dev-server <source folder name>
```

**Review by ebrandin — CHANGES_REQUESTED** [review]
Readme file could be improved

**Pull request merged**